Infrastructure-Scripts/Import-ApimSwaggerApiDefinition.ps1

```diff
@@ -176,14 +198,39 @@ foreach ($SwaggerPath in $SwaggerPaths) {
         throw "Failed to import API definition after $ImportRetries attempts"
     }
 
+    Remove-Item -Path $SwaggerSpecificationFilePath
+
     Add-AzApiManagementApiToProduct -Context $Context -ProductId $ProductId -ApiId $ApiId
-    if ($SandboxProductId) {
-        Write-Verbose "Adding API to sandbox product"
-        Add-AzApiManagementApiToProduct -Context $Context -ProductId $SandboxProductId -ApiId $ApiId
-    }
 
     Set-AzApiManagementPolicy -Context $Context -ApiId $ApiId -Policy $PolicyString
 }
 
+$SwaggerSpecificationFilePath = "./swagger-specification.json"
```

Comment:
Does this not need be defined earlier in the script?

Reply:
It's used later when the Import-Api function is called